python/py-gsd/Portfile

```diff
@@ -4,13 +4,13 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        glotzerlab gsd 1.8.1 v
+github.setup        glotzerlab gsd 2.1.2 v
 name                py-gsd
 categories-append   science
 platforms           darwin
 license             BSD
 
-python.versions     27 36 37
+python.versions     27 36 37 38
 python.default_version 37
```

Comment:
is this necessary or can you just let the `python` PG pick the value?

Reply:
Definitely. I actually didn't know you can use the port group to pick the value. I guess I inherited this line from somewhere. I'll fix it thanks!